gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -185,10 +185,10 @@ public ZoneInfo build() {
   }
 
   /**
-   * Returns a builder for {@code ZoneInfo} with an assigned {@code name}.
+   * Returns a ZoneInfo object with assigned {@code name}, {@code dnsName} and {@code description}.
    */
-  public static Builder builder(String name) {
-    return new BuilderImpl(name);
+  public static ZoneInfo of(String name, String dnsName, String description) {
```

Comment:
I checked and they're all mandatory. Weird that `description` is a mandatory field. Sorry, forget this comment.